float64 AH
float64 AV
float64 B1
float64 B2
float64 B3
float64 CU
float64 CD
float64 CL
float64 CR
